plugin/cshared.go

```diff
@@ -0,0 +1,284 @@
+package plugin
+
+/*
+#include <stdlib.h>
+*/
+import "C"
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"reflect"
+	"time"
+	"unsafe"
+
+	"github.com/fluent/fluent-bit-go/input"
```

Comment:
aren't we referring to calyptia?

Reply:
That's a good question. This package is still named `github.com/fluent/fluent-bit-go`. See https://github.com/calyptia/fluent-bit-go/blob/b3af4fd037f50d243b0b5e61ca5834cf3b36e70c/go.mod#L1

Shall I rename it? 🤔 

Will we continue on this fork as a separate thing, or will we merge it to the fluent one at some point?